### ISSUE TITLE
improve hyperparameters exploring notebook

### ISSUE THREAD

**Issue by chenhuims** [microsoft/computervision-recipes#186]
Please find a few comments below for improving this [notebook](https://github.com/microsoft/ComputerVision/blob/master/classification/notebooks/11_exploring_hyperparameters.ipynb):
* Change "Testing parameter" in Table of Contents to "Testing hyperparameters".
* "Lets say we want to learn more about how different learning rates and different image sizes affect our model's accuracy when restricted to 10 epochs, and we want to build an experiment to test out these hyperparameters." where "Lets" should be "Let's". In addition, it may be worth explaining the importance of exploring/tuning the hyperparameters.
* Consider to specify the modules explicitly in 
```from utils_cv.classification.parameter_sweeper import *```
* "We want to see whether the original images (which are watermarked) will perform just as well as the non-watermarked images." why this is important?
* "Before we start testing, it's a good idea to see what the default parameters Are" where "Are" should be "are".
* In cell 6, it might be better to describe the meaning of each hyperparameter in a table.
